datamodels/2.x/authent-external/dictionaries/en_gb.dict.authent-external.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Localized data
+ *
+ * @copyright Copyright (C) 2024 Combodo SAS
+ * @license    http://opensource.org/licenses/AGPL-3.0
```

Comment:
Misaligned URL, I would just all have only a single space:
```suggestion
 * @copyright Copyright (C) 2024 Combodo SAS
 * @license https://opensource.org/licenses/AGPL-3.0
```
Maybe Combodo will ask to align them..
I also think the URL needs to be https.